### ISSUE TITLE
Stratego 11 says "support not implemented" for learning queries, when runing verifyta from command line

### ISSUE THREAD

**Issue by AstridHornBrorholt** [UPPAALModelChecker/UPPAAL-Meta#197]
**Describe the bug**

I wanted to upgrade my experiments to use Stratego 11, but it errors on `minE` queries, when I run it from the command line. I have created a small example query (see To Reproduce) 

`strategy S = minE(x) [<=1000] {Template.location} -> {}: <> time >= 1000`

Running it from the GUI is still fine, but when I invoke `uppaal-4.1.20-stratego-11-linux64/bin/verifyta`, it gives the following message:

```
➜  ~ /home/asger/opt/uppaal-4.1.20-stratego-11-linux64//bin/verifyta learningTest.xml learningTest.q
Options for the verification:
  Generating no trace
  Search order is breadth first
  Using conservative space optimisation
  Seed is 1681203119
  State space representation uses minimal constraint systems with future testing
  Using HashMap + Compress integers for discrete state storage

Verifying formula 1 at learningTest.q:6
              -- Aborted.
learningTest.xml::7: [error] Support is not implemented for the given expression type.
```

However, this was not an issue in the previous version:

```
➜  ~ /home/asger/opt/uppaal-4.1.20-stratego-10-linux64//bin/verifyta learningTest.xml learningTest.q
Options for the verification:
  Generating no trace
  Search order is breadth first
  Using conservative space optimisation
  Seed is 1681203128
  State space representation uses minimal constraint systems with future testing
  Using HashMap + Compress integers for discrete state storage

Verifying formula 1 at learningTest.q:6
 -- Formula is satisfied.
(9000 runs)
Learning statistics for best strategy: 
	Number of resets: 0
	Number of iterations in last reset: 1
	Number of iterations in total: 1
```

**To Reproduce**

Simple example: 

![image](https://user-images.githubusercontent.com/11016262/231111475-cf7c2545-734a-4e91-9187-8965f3ffdd50.png)

[learningTest.zip](https://github.com/UPPAALModelChecker/UPPAAL-Meta/files/11198664/learningTest.zip)

```
/path/to/uppaal-4.1.20-stratego-11-linux64//bin/verifyta learningTest.xml learningTest.q
/path/to/uppaal-4.1.20-stratego-10-linux64//bin/verifyta learningTest.xml learningTest.q
```

**Versions of UPPAAL tested**

```
UPPAAL 4.1.20-stratego-11 (rev. 323D987339A98B21), December 2022
UPPAAL 4.1.20-stratego-10 (rev. 37ECAAA437FE60EE), October 2022
```

**Desktop**
Ubuntu 22.04.2 LTS

openjdk 11.0.18 2023-01-17
OpenJDK Runtime Environment (build 11.0.18+10-post-Ubuntu-0ubuntu122.04)
OpenJDK 64-Bit Server VM (build 11.0.18+10-post-Ubuntu-0ubuntu122.04, mixed mode, sharing)

**Comment by thorulf4** [comment]
While we are trying to fix this issue you can circumvent it by not using a query file `.q` as the issue is specific to query file handling.
`verifyta learningTest.xml learningTest.q` => `verifyta learningTest.xml`
This works because the query is also stored inside the `learningTest.xml` file
Sorry for the inconvenience

**Comment by thorulf4** [comment]
This issue was resolved in In Uppaal 5.0
